### PR TITLE
Adding custom labels in query and collector ingresses

### PR DESCRIPTION
**Pull request by NicolasLemince**
Signed-off-by: Nicolas <nicorabatel@gmail.com>

#### What this PR does
Adding values to be able to add custom labels in query and collector ingresses

#### Which issue this PR fixes

- fixes https://github.com/jaegertracing/helm-charts/issues/104

#### Checklist

- [ ] [DCO](https://github.com/jaegertracing/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [ ] Commits are [GPG signed](https://docs.github.com/en/github/authenticating-to-github/about-commit-signature-verification)
- [ ] Chart Version bumped
- [ ] Title of the PR starts with chart name (`[jaeger]` or `[jaeger-operator]`)
- [ ] README.md has been updated to match version/contain new values

### CODE REVIEW

File: charts/jaeger/templates/collector-ing.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ kind: Ingress
 metadata:
   name: {{ template "jaeger.collector.name" . }}
   labels:
-    {{- include "jaeger.labels" . | nindent 4 }}
+    {{- toYaml .Values.collector.ingress.labels | nindent 4 }}
```
Comment (naseemkullah) [inline charts/jaeger/templates/collector-ing.yaml:10]:
```suggestion
    {{- include "jaeger.labels" . | nindent 4 }}
    {{- if .Values.query.ingress.labels }}
    {{- toYaml .Values.collector.ingress.labels | nindent 4 }}
    {{- end }}
```

what about something like this that keeps the common labels, but allows folks to add additional labels to the ingress?

File: charts/jaeger/templates/query-ing.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@ kind: Ingress
 metadata:
   name: {{ template "jaeger.query.name" . }}
   labels:
-    {{- include "jaeger.labels" . | nindent 4 }}
+    {{- toYaml .Values.query.ingress.labels | nindent 4 }}
```
Comment (naseemkullah) [inline charts/jaeger/templates/query-ing.yaml:10]: same as above

**Review by naseemkullah — CHANGES_REQUESTED** [review]
I propose that rather than replace the labels, add an option to append labels - would that work for you?

File: charts/jaeger/templates/collector-ing.yaml (hunk 1 above)
Comment (NicolasLemince) [reply charts/jaeger/templates/collector-ing.yaml:12]: Totally agree, this is done.

File: charts/jaeger/templates/query-ing.yaml (hunk 2 above)
Comment (NicolasLemince) [reply charts/jaeger/templates/query-ing.yaml:12]: Totally agree, this is done.

**Comment by NicolasLemince** [comment]
That's ok, I made the changes + I signed the last commit

**Review by naseemkullah — APPROVED** [review]

**Pull request closed without merging**